src/Search/Search.tsx

```diff
@@ -1,5 +1,37 @@
-import React from "react";
-import * as ReactDOM from "react-dom";
+import React, { useState, useEffect } from "react";
 import './Search.scss'
 
+
+const Search: React.FC = () => {
+  const [query, setQuery] = useState<string>('')
+  const [debouncedQuery, setDebouncedQuery] = useState<string>(query)
+
+  useEffect(() => {
+    const timerId = setTimeout(() => {
+      setDebouncedQuery(query)
```

Comment:
Should we start with the search functionality being a drop-down of all supported cities? We could then add functionality that would locate the nearest supported city and display that as a stretch goal?